packages/utils/scrollbar-width.ts

```diff
@@ -22,7 +22,7 @@ export default function (): number {
   outer.appendChild(inner)
 
   const widthWithScroll = inner.offsetWidth
-  outer.parentNode.removeChild(outer)
+  outer.parentNode!.removeChild(outer)
```

Comment:
I think the use of non-nullish operator should be very careful which could cause trouble. I suggest that we use `optional` chaining operator here instead of `non-nullish assertion` to prevent error.

Reply:
Yes, you are right.